ios/browser/api/networking/models/extensions/BraveCertificatePKIXOCSPCRLIDExtensionModel.mm

```diff
@@ -0,0 +1,31 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#import "BraveCertificatePKIXOCSPCRLIDExtensionModel.h"
+#if defined(BRAVE_CORE) // Compiling in Brave-Core
+  #import "brave/ios/browser/api/networking/common/brave_certificate_enums.h"
+  #include "brave/ios/browser/api/networking/utils/brave_certificate_ios_utils.h"
+  #include "brave/ios/browser/api/networking/utils/brave_certificate_utils.h"
+#else
+  #import "brave_certificate_enums.h"
+  #include "brave_certificate_ios_utils.h"
+  #include "brave_certificate_utils.h"
+#endif
+
+#if defined(BRAVE_CORE) // Compiling in Brave-Core
+  #include "third_party/boringssl/src/include/openssl/x509.h"
+  #include "third_party/boringssl/src/include/openssl/x509v3.h"
+#else
+  #include <openssl/x509.h>
+  #include <openssl/x509v3.h>
+#endif
+
+
+@implementation BraveCertificatePKIXOCSPCRLIDExtensionModel
+- (void)parseExtension:(X509_EXTENSION*)extension {
+  //OCSP_CRLID structure not exposed..
+  //OCSP_CRLID* crlId = static_cast<OCSP_CRLID*>(X509V3_EXT_d2i(extension));
```

Comment:
What does this mean for the `nonce` property defined? Always nil/empty-string? You haven't initialized it like you've done in the past extensions

Reply:
Fixed. This extension isn't used yet because the API isn't internally exposed. I added an assertion and comment for future.